package.json

```diff
@@ -91,5 +91,8 @@
     "tabWidth": 4,
     "jsxSingleQuote": true,
     "singleQuote": true
+  },
+  "dependencies": {
+    "react-infinite-scroller": "^1.2.4"
```

Comment:
This file controls parts of our admin UI, so it doesn't have an impact on `EntitiesList`. Please revert the change, as well as changes to `yarn.lock`

Reply:
Hey @mathjazz , i can't seem to pass through two test cases(as attached in the image below) due to the fact that in the EntitiesList.test.js wrapper.instance().anyfn() returns null for stateless components,so i can't access the function getMoreEntities or toggleForBatchEditing,i have tried using forwardRefs to expose the function(in this case getMoreEntities), it was able to refer to it but, i still can't seem to get it to  access the actions property(giving me TypeError: actions is undefined), maybe you can help me out with your suggestions, i have also attached (getMoreEntities.png) how i modified the test. 

![testCases](https://user-images.githubusercontent.com/38439166/151139944-76c2decc-09aa-4d40-8640-a9291e79bbf9.png)
![getMoreEntities](https://user-images.githubusercontent.com/38439166/151142776-6e5348b4-a402-4f38-8a44-953c96ef71eb.png)



**PS: This is my first time contributing to an open source project, so kindly take this as a way of me learning from you. : )**
.